tests/basic/pool_function_test.go

```diff
@@ -994,3 +995,127 @@ var _ = Describe("{PoolExpandTestLimits}", func() {
 		dash.VerifyFatal(err != nil, true, "DMThin pool expansion to 20 TB should result in error")
 	})
 })
+
+var _ = Describe("{PoolExpandAndCheckAlertsUsingAddDisk}", func() {
+
+	var testrailID = 34542894
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542894
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolExpandAndCheckAlertsUsingAddDisk", "pool expansion using add-disk and check alerts after each operation", nil, testrailID)
+		contexts = scheduleApps()
```

Comment:
After this command, Lets validate that scheduled Apps come up successfully using ValidateApplications API.
https://github.com/portworx/torpedo/blob/master/tests/common.go#L1891

Reply:
fixed